dissect/cstruct/types/enum.py

```diff
@@ -84,6 +85,19 @@ def _write_0(cls, stream: BinaryIO, array: list[BaseType]) -> int:
         data = [entry.value if isinstance(entry, Enum) else entry for entry in array]
         return cls._write_array(stream, data + [cls.type()])
 
+    def _class_stub(cls) -> str:
+        return f"class {cls.__name__}({cls.__base__.__name__}, {cls.type.__name__}):\n"
+
+    def to_type_stub(cls, name: str = "") -> str:
+        output = ""
+        with io.StringIO() as buf:
+            buf.write(cls._class_stub())
+            for key in cls.__members__.keys():
+                buf.write(f"    {key} = ...\n")
+            output = buf.getvalue()
+
+        return output
```

Comment:
```suggestion
        result = [cls._class_stub()]
        for key in cls.__members__.keys():
            result.append(f"    {key} = ...")
        return "\n".join(result)
```
Should also get rid of the newline in `_class_stub`. Might be confusing for custom type implementations that you're responsible for ending with a newline.